cpp/tests/test_abm_person.cpp

```diff
@@ -330,3 +325,56 @@ TEST(Person, rng)
     ASSERT_EQ(p.get_rng_counter(), mio::Counter<uint32_t>(1));
     ASSERT_EQ(p_rng.get_counter(), mio::rng_totalsequence_counter<uint64_t>(13, mio::Counter<uint32_t>{1}));
 }
+
+TEST(TestPerson, applyTestIntervention)
+{
+    using testing::Return;
+    auto rng = mio::RandomNumberGenerator();
+    ScopedMockDistribution<testing::StrictMock<MockDistribution<mio::UniformDistribution<double>>>>
+        mock_uniform_dist_antigen;
+    EXPECT_CALL(mock_uniform_dist_antigen.get_mock(), invoke)
+        .Times(6)
+        .WillOnce(Return(0.8))
+        .WillOnce(Return(0.8))
+        .WillOnce(Return(0.8))
+        .WillOnce(Return(0.8))
+        .WillOnce(Return(0.8))
+        .WillOnce(Return(0.8));
+
+    mio::abm::Location home(mio::abm::LocationType::Home, 0, num_age_groups);
+    mio::abm::Location target(mio::abm::LocationType::Work, 0, num_age_groups);
+    auto person     = make_test_person(home);
+    auto rng_person = mio::abm::Person::RandomNumberGenerator(rng, person);
+
+    person.set_compliance(mio::abm::InterventionType::Testing, 1);
+    ASSERT_TRUE(person.is_compliant(rng_person, mio::abm::InterventionType::Testing));
+
+    person.set_compliance(mio::abm::InterventionType::Testing, 0.4);
+    ASSERT_FALSE(person.is_compliant(rng_person, mio::abm::InterventionType::Testing));
+}
+
+TEST(TestPerson, applyIsolationIntervention)
```

Comment:
Same as applyTestIntervention. Since the tests are exactly the same, consider making a [parametrized test](https://google.github.io/googletest/advanced.html#value-parameterized-tests)

Reply:
See above.